### PR TITLE
Use skip range

### PR DESCRIPTION
**Pull request by rporres**
From #56 last comments

### CODE REVIEW

**Review by rrati — APPROVED** [review]
/lgtm

File: deploy/bundle/generate-csv-template.py [hunk 1]
```diff
@@ -58,10 +58,8 @@
     csv['spec']['replaces'] = f'deployment-validation-operator.v{args.replaces}'
 
 if args.skip:
-    csv['spec']['skips'] = [
-        f'deployment-validation-operator.v{version}' for version in
-        sorted(args.skip, key=lambda v: int(v.split('.')[2].split('-')[0]))
-    ]
+    csv['metadata']['annotations']['olm.skipRange'] = ' || '.join(
```
Comment (2uasimojo) [inline deploy/bundle/generate-csv-template.py:61]: ✓

Comment (rporres) [reply deploy/bundle/generate-csv-template.py:61]: They are the semver versions, not the full csv name. That's what we had before.

Comment (2uasimojo) [reply deploy/bundle/generate-csv-template.py:61]: Nice.

So... don't we also need to set the `version` field?

Comment (rrati) [reply deploy/bundle/generate-csv-template.py:61]: Yes.  Version field needs to be set for skipRange to work.

Comment (rporres) [reply deploy/bundle/generate-csv-template.py:61]: Version in the csv: https://github.com/app-sre/deployment-validation-operator/blob/master/deploy/bundle/template/deploymentvalidationoperator.clusterserviceversion.yaml#L63

And it's passed via `oc process` https://github.com/app-sre/deployment-validation-operator/blob/master/hack/build_channel_catalog.sh#L136

Comment (2uasimojo) [reply deploy/bundle/generate-csv-template.py:61]: Ah, cool, I didn't look far enough to see that bit. Thanks!

Comment (rporres) [reply deploy/bundle/generate-csv-template.py:61]: yw!

**Pull request merged**